### PR TITLE
utils/chunked_vector: fix a typo in comment

### PR DESCRIPTION
**Pull request by tchaikov**
- [x] no need to backport. typo in comment

### CODE REVIEW

**Review by denesb — APPROVED** [review]

File: utils/chunked_vector.hh [hunk 1]
```diff
@@ -471,7 +471,7 @@ template <typename T, size_t max_contiguous_allocation>
 void chunked_vector<T, max_contiguous_allocation>::set_chunk_after_emplace_back(std::pair<chunk_ptr, size_t> x) noexcept {
     auto new_chunk_ptr = std::move(x.first);
     auto new_chunk_capacity = x.second;
-    // If the new chunk os replacing the first chunk, migrate the existing elements onto it.
+    // If the new chunk is replacing the first chunk, migrate the existing elements onto it.
```
Comment (denesb) [inline utils/chunked_vector.hh:474]: @tchaikov while at it, can you also fix the `capcity` mispelling below?

Comment (tchaikov) [reply utils/chunked_vector.hh:474]: yeah, thanks! fixed and repushed.

**Comment by tchaikov** [comment]
v2:

- s/capcity/capacity/

**Review by bhalevy — APPROVED** [review]
LGTM, thanks!

**Comment by denesb** [comment]
Merging with failed CI because I know for sure, that two typo fixes in the comment didn't cause the CI failure.
@tchaikov please annotate the CI failure though.

**Comment by tchaikov** [comment]
> * [topology_experimental_raft.test_tablets.debug.5](https://jenkins.scylladb.com//job/scylla-master/job/scylla-ci/8455/testReport/junit/%28root%29/non-boost%20tests/Tests___Unit_Tests___topology_experimental_raft_test_tablets_debug_5) [🔍](https://github.com/scylladb/scylladb/issues?q=is:issue+is:open+topology_experimental_raft.test_tablets.debug.5)

https://github.com/scylladb/scylladb/issues/18492

**Pull request closed without merging**